pkg/api/api.go

```diff
@@ -103,6 +103,26 @@ type IssueCodeResponse struct {
 	ErrorCode          string `json:"errorCode,omitempty"`
 }
 
+// CheckCodeStatusRequest defines the parameters to request the status for a
+// previously issued OTP code. This is called by the Web frontend.
+// API is served at /api/codestatus
+type CheckCodeStatusRequest struct {
+	ID []string `json:"id"`
```

Comment:
might be good to ask the issue filer if they need the ability to do this by Array?

maybe one at a time is good enough?

Reply:
one at a time is certainly easier. The issue filer mentioned automation which might involve multiple queries (eg a health provider checking the status for their patients for follow-up). I'll reply in the issue.

Reply:
The user notes they only need one